src/pages/login/login.page.tsx

```diff
@@ -0,0 +1,60 @@
+import { useState } from 'react';
+import { Form, Input, Button, message } from 'antd';
+import axios from 'axios';
+
+import './login.scss';
+
+function LoginForm() {
+  const [loading, setLoading] = useState(false);
+
+  const handleSubmit = async (values: any) => {
+    setLoading(true);
+
+    try {
+      const response = await axios.post('/login', values);
+      console.log(response.data);
```

Comment:
Please don't push any debug statements to the codebase, it clutters the terminal.